cli/dcoscli/data/help/node.txt

```diff
@@ -13,7 +13,8 @@ Usage:
     dcos node list-components [--leader --mesos-id=<mesos-id> --json]
     dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<agent-id>]
                   [--component=<component-name> --filter=<filter>...]
-    dcos node metrics (--mesos-id=<mesos-id>) [--filter=<name>... | --json]
+    dcos node metrics details (--mesos-id=<mesos-id>) [--json]
```

Comment:
since mesos-id is required this is: `dcos node metrics details <mesos-id> [--json]`